workloads/network-perf/run_network_compare.sh

```diff
@@ -9,38 +9,38 @@ if [ "$#" -ne 2 ]; then
 fi
 base_uuid=$1
 compare_uuid=$2
-es=${ES_SERVER}
-if [ "$es" == "" ]; then
-  echo "Unable to execute compare - no ES passed"
+es_server=${ES_SERVER}
+es_port=${ES_PORT}
+if [ "$es_server" == "" ] || [ "$es_port" == "" ]; then
+  echo "Unable to execute compare - no elasticsearch server and/or port passed"
   exit 1
 fi
 
 git clone https://github.com/cloud-bulldozer/touchstone
 cd touchstone
-python -m venv ./compare
+python3 -m venv ./compare
 source ./compare/bin/activate
-pip install -r requirements.txt
+pip3 install -r requirements.txt
 python3 setup.py develop
 if [[ $? -ne 0 ]] ; then
   echo "Unable to execute compare - Failed to install touchstone"
   exit 1
 fi
 
-touchstone_compare $tool $datasource ripsaw -url $es -u $base_uuid $compare_uuid -o yaml | tee ../compare.yaml
+touchstone_compare $tool $datasource ripsaw -url $es_server:$es_port -u $base_uuid $compare_uuid -o yaml | tee ../compare.yaml
```

Comment:
for an artifact , could we generate the csv (I need to get that added to touchstone, I already have a PR to send..)